Source/BlockchainService/ChainpointService.swift

```diff
@@ -66,8 +68,66 @@ final class ChainpointService: BlockchainService {
                                submittedHashes: [],
                                completion: completion)
     }
+    
+    // MARK: Proof
+    func proof(forHashId: HashIdNode, completion: @escaping (Result<Proof>) -> Void) {
 
-    private func submitHashRequest(_ request: SubmitHashRequest?,
+        // TODO: David Szurma - Should discuss the endpoint structure
+        let url = URL(string: "http://35.230.179.171")!
```

Comment:
Yeah, like I mentioned above, we should only support NodeHash instances to ask for proofs. This way this problem would be solved I think.